### ISSUE TITLE
How to retrieve time series data from AQUARIUS with R

### ISSUE THREAD

**Issue by WatershedFlow** [AquaticInformatics/getting-started#7]
Hi there,

I am using AQUARIUS system in my work. Before this new update just installed recently (i.e., Jan 2020) we used GetTimeSeriesData to retrieve time series data which is in csv format. 

Then with this new update, Jason file is used in AQUARIUS. Do you have some example R codes to retrieve AQUARIUS time series data as we did with GetTimeSeriesData?

West

**Comment by DougSchmidt-AI** [comment]
Yes, we have R examples available [here](https://github.com/AquaticInformatics/examples/tree/master/TimeSeries/PublicApis/R).

These examples all use the [`timeseries_client.r` API wrapper code](https://github.com/AquaticInformatics/examples/tree/master/TimeSeries/PublicApis/R#the-timeseries_clientr-api-wrapper) , which handles the HTTP requests and JSON serialization for you. And many of the APIs exposed in the wrapper will work on both the legacy AQUARIUS Time-Series 3.x versions, as well as the more recent 20xx versions.
